app/models/enrollment.rb

```diff
@@ -0,0 +1,10 @@
+class Enrollment < ActiveRecord::Base
```

Comment:
It is. I tried switching to `GameEnrollment`, which is a bit more descriptive, but the code looks much cleaner with `Enrollment`.

I think it's safe to assume in this scenario what's an enrollment, but if everyone feels different, I'd be happy to adjust.